buildspec-build.yaml

```diff
@@ -0,0 +1,20 @@
+version: 0.2
+phases:
+  install:
+    commands:
+      - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2&
+      - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
+  pre_build:
+    commands:
+      -  aws ecr get-login-password --region $REGION | docker login --username AWS --password-stdin 774051255656.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
I assume `774051255656.dkr.ecr.us-east-1.amazonaws.com` represents the ECR repo we want to push to?  If so, this won't be the same for every lab, so we'll need to figure out how to set this as a variable.

Reply:
That's true, I can set the product_image_repo variable inside the environment of codebuild and just use it in the buildspec